### DISCUSSION TITLE
typeorm-save-options/

### DISCUSSION THREAD

**Discussion by giscus** [JHyeok/JHyeok.com#55] [Comments]
# typeorm-save-options/

주로 Sequelize만 사용하다가 우연히 TypeORM에서 데이터베이스에 실행하는 쿼리들을 확인하고 SaveOptions으로 테스트를 했습니다. 아래의 테스트에 사용된 Sequelize의 버전은 6.28.0(@nestjs/sequelize v9.0.0), TypeORM의 버전은 0.…

https://jhyeok.com/typeorm-save-options/

**Comment by MinJungHyun** [comment]
감사합니다.
헷갈릴떄마다 찾아 봐야겠네요
